core/types/crosslink.go

```diff
@@ -4,51 +4,62 @@ import (
 	"math/big"
 	"sort"
 
-	"github.com/ethereum/go-ethereum/rlp"
-
 	"github.com/ethereum/go-ethereum/common"
-
+	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/harmony-one/harmony/block"
 )
 
 // CrossLink is only used on beacon chain to store the hash links from other shards
+// signature and bitmap correspond to |blockNumber|parentHash| byte array
+// Captial to enable rlp encoding
 type CrossLink struct {
-	ChainHeader *block.Header
```

Comment:
where is the written proof/explanation why possible to have same level of verification without full Header? Needs to be written out, explained

Reply:
We don't have same level of verification. Cannot verify the block header content, but trust the majority of the committee is not corrupted. I will write some comments